packages/nuxt/src/core/app.ts

```diff
@@ -157,7 +159,7 @@ export async function resolveApp (nuxt: Nuxt, app: NuxtApp) {
 
```

Comment:
@danielroe in the 2 lines above, is the re-deduplication of the middleware and the plugins needed in lines 171-172?

Reply:
It is there because users may add their own plugins or middleware on L170.

Reply:
I see, I'll leave it be then.